plugins/go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/vmware-tanzu/antrea/plugins
+
+go 1.13
+
+require (
+	github.com/vmware-tanzu/antrea v0.7.1
+	github.com/vmware-tanzu/octant v0.13.1
+	k8s.io/apimachinery v0.19.0-alpha.3
+	k8s.io/client-go v0.19.0-alpha.3
+)
+
+replace (
+	github.com/contiv/ofnet => github.com/wenyingd/ofnet v0.0.0-20200601065543-2c7a62482f16
+	github.com/vmware-tanzu/antrea => ../
+	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20200403204345-e1beb1bd0f35
```

Comment:
could you add a comment on why this one is needed?

Reply:
OK, go it.